__tests__/index.test.js

```diff
@@ -262,3 +261,37 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe('POST /api/articles/:article_id/comments', () => {
+  it('POST 201: adds comment to an article and responds with the posted comment', () => {
+    const inputBody = {
+      username: "lurker",
+      body: "Excellent work from the author"
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(inputBody)
+      .expect(201)
+      .then(({ body }) => {
+        const { newComment } = body;
+        const expected = {
+          body: "Excellent work from the author",
+          article_id: 2,
+          author: 'lurker'
```

Comment:
It would be good if in this test you check the comment looks how you have specified in the example response, there are a few keys you haven't included. Where you can't be sure of the exact content, you can check the datatype 🔍 
```js 
   "comment_id": 19,
         "body": "Excellent work from the author",
          "article_id": 2,
          "author": "lurker",
          "votes": 0,
          "created_at": "2020-11-24T00:08:00.000Z"
```